src/transforms/pipelines/mod.rs

```diff
@@ -359,4 +363,49 @@ mod tests {
             vec!["foo.type_router._unmatched", "foo.logs.1"]
         );
     }
+
+    #[tokio::test]
+    async fn check_compliance() {
+        use crate::event::LogEvent;
+
+        assert_transform_compliance(async move {
+            let config = toml::from_str::<PipelinesConfig>(
+                r#"
+[transforms.my_pipelines]
+type = "pipelines"
+inputs = ["syslog"]
+
+[[transforms.my_pipelines.logs]]
+name = "foo pipeline"
+
+[[transforms.my_pipelines.logs.transforms]]
+
+[[transforms.my_pipelines.logs]]
+name = "bar pipeline"
+filter.type = "vrl"
+filter.source = """true"""
+
+
+
+            "#,
+            )
```

Comment:
Is that config valid / is this test ever executed? Asking since I don't see any `syslog` component that is referenced in `inputs` here.

(Also could be formatted a bit more nicely using `indoc`.)